### ISSUE TITLE
Wrong release? "Cannot find module 'extract-text-webpack-plugin'"

### ISSUE THREAD

**Issue by LeoDT** [insin/nwb#306]
This issue is a: 

- [x] Bug report
- [ ] Feature request
- [ ] Question / support request
- [ ] Other

```
Error running command: Cannot find module 'extract-text-webpack-plugin'
Error: Cannot find module 'extract-text-webpack-plugin'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/xxx/node_modules/nwb/lib/createWebpackConfig.js:33:33)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/xxx/node_modules/nwb/lib/createServerWebpackConfig.js:9:28)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
```
createWebpackConfig.js line 33:
```
var _extractTextWebpackPlugin = require('extract-text-webpack-plugin');
```

should be '@insin/extract-text-webpack-plugin'?

**Comment by loklaan** [comment]
Thank's for filing an issue and picking this up!

@insin If you inspect the `0.15.7` tgz, you'll see that `createWebpackConfig.js` was built without the correct extract-text dep. Any idea how this could have happened?

**Comment by insin** [comment]
Should be fixed by 35fe4cf07a, will release a fix this evening
